pyproject.toml

```diff
@@ -24,7 +24,8 @@ dependencies = [
     'pandas',
     'pyserial',
     "pyOSC3@git+https://github.com/glopesdev/pyosc3.git@master",
-    "newscale@git+https://github.com/AllenNeuralDynamics/python-newscale@axes-on-target"
+    "newscale@git+https://github.com/AllenNeuralDynamics/python-newscale@axes-on-target",
+    "aind-auto-train@git+https://github.com/AllenNeuralDynamics/aind-foraging-behavior-bonsai-automatic-training.git@main"
```

Comment:
Can you also add this dependency to the requirements.txt

Reply:
added.